src/main/webapp/WEB-INF/tags/instructor/studentsList/searchBox.tag

```diff
@@ -0,0 +1,36 @@
+<%@ tag description="instructorStudentList - Student search box" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ attribute name="data" type="teammates.ui.template.InstructorStudentsListSearchBox" required="true" %>
+<c:set var="TOOLTIP_SEARCH_STUDENT" value="<%= Const.Tooltips.SEARCH_STUDENT %>" />
+<c:set var="SEARCH_KEY" value="<%= Const.ParamsNames.SEARCH_KEY %>" />
+<c:set var="SEARCH_STUDENTS" value="<%= Const.ParamsNames.SEARCH_STUDENTS %>" />
+<c:set var="SEARCH_COMMENTS_FOR_STUDENTS" value="<%= Const.ParamsNames.SEARCH_COMMENTS_FOR_STUDENTS %>" />
+<c:set var="SEARCH_COMMENTS_FOR_RESPONSES" value="<%= Const.ParamsNames.SEARCH_COMMENTS_FOR_RESPONSES %>" />
+<c:set var="USER_ID" value="<%= Const.ParamsNames.USER_ID %>" />
```

Comment:
It's okay to use scriptlets directly for constants in the HTML.
Setting is only necessary if you want to use the constant in an EL expression.